File: app/src/main/java/cn/intret/app/picgo/ui/main/MoveFileResultMessage.java
package cn.intret.app.picgo.ui.main;

import java.io.File;

import cn.intret.app.picgo.model.MoveFileResult;

class MoveFileResultMessage {
    private MoveFileResult mResult;
    private File mDestDir;

    public MoveFileResultMessage setResult(MoveFileResult result) {
        mResult = result;
        return this;
    }

    public MoveFileResult getResult() {
        return mResult;
    }

    public MoveFileResultMessage setDestDir(File destDir) {
        mDestDir = destDir;
        return this;
    }

    public File getDestDir() {
        return mDestDir;
    }
}

File: app/src/main/java/cn/intret/app/picgo/screens/event/MoveFileResultMessage.kt
package cn.intret.app.picgo.screens.event

import cn.intret.app.picgo.model.image.data.MoveFileResult
import java.io.File

class MoveFileResultMessage {
    private var mResult: MoveFileResult? = null
    private var mDestDir: File? = null

    fun setResult(result: MoveFileResult): MoveFileResultMessage {
        mResult = result
        return this
    }

    fun getResult(): MoveFileResult? {
        return mResult
    }

    fun setDestDir(destDir: File): MoveFileResultMessage {
        mDestDir = destDir
        return this
    }

    fun getDestDir(): File? {
        return mDestDir
    }
}
